### DISCUSSION TITLE
i18n

### DISCUSSION THREAD

**Discussion by jalenzz** [plausible/analytics#868] [Anything Else]
Is it possible to support more language like Chinese on the showcase page?

**Comment by mzalevski** [comment]
It would be ideal for the stats page on my website about Polish constitution: [konstytucja.online/statystyki](https://konstytucja.online/statystyki). I will gladly provide all the `pl` translations.

**Comment by milotype** [comment]
I also think it would be great if **Plausible Analytics** could be translated/localized into different languages. If you decide upon it, please take special attention to plural forms, decimal delimiters, etc. …

It looks and feels odd, when viewing a webpage in some other language then english, suddenly to see english strings/phrases in an analytics view.

I'd like to suggest to use a translation platform for translations – my favorite is Weblate (but there are others of course) – hosted, self-hosted, freee, paid … whatever suites youer needs. ;-)

**Comment by milotype** [comment]
Also, if possible, enable translations for the app, as well as for the website …

**Comment by bfontaine** [comment]
I’m willing to help on French translations as well, if there’s a need.

**Reply by comradekingu** [reply]
Happy to help generally and for nb_NO if you use Weblate. The "Hosted" platform uses self-hosted Matomo btw.

Came here to help translate the project, and found I can't do so easily.
Translating https://hosted.weblate.org/projects/matomo/ is taking weeks,
and improving the source strings for every plugin will take months.
